pkg/restapi/diddochandler/restapi_test.go

```diff
@@ -43,6 +44,9 @@ func TestRESTAPI(t *testing.T) {
 	s.start()
 	defer s.stop()
 
+	// Wait a second for the service to start
+	time.Sleep(time.Second)
```

Comment:
second may not be enough, you should poll until server becomes available or you reach timeout

Reply:
Done